src/createServer.js

```diff
@@ -1,7 +1,47 @@
 'use strict';
 
+const express = require('express');
+const {
+  getUsers,
+  getUser,
+  deleteUser,
+  updateUser,
+  postUser,
+} = require('./controllers/users-controller.js');
+const {
+  getExpenses,
+  postExpense,
+  getExpense,
+  deleteExpense,
+  patchExpense,
+} = require('./controllers/expenses-controller.js');
+
 const createServer = () => {
-  // your code goes here
+  const app = express();
+
+  // Users
+  app.get('/users', express.json(), getUsers);
+
+  app.post('/users', express.json(), postUser);
+
+  app.get('/users/:userId', express.json(), getUser);
+
+  app.delete('/users/:userId', express.json(), deleteUser);
+
+  app.patch('/users/:userId', express.json(), updateUser);
+
+  // Expenses
+  app.get('/expenses', express.json(), getExpenses);
+
+  app.post('/expenses', express.json(), postExpense);
+
+  app.get('/expenses/:expenseId', express.json(), getExpense);
+
+  app.delete('/expenses/:expenseId', express.json(), deleteExpense);
+
+  app.patch('/expenses/:expenseId', express.json(), patchExpense);
```

Comment:
Move express.json() middleware outside individual route handlers and apply it globally to the app.  `app.use(express.json());`